### ISSUE TITLE
Add support for Java 9!

### ISSUE THREAD

**Issue by aadrian** [JFormDesigner/markdown-writer-fx#25]
Please add support for Java 9 too.

The Travis-CI relevant part of the build matrix for Java 9:
```yml
    - dist: trusty
      group: edge
      jdk: oraclejdk9
      sudo: required
      addons:
        apt:
          packages:
            - oracle-java9-installer
```

Thank you.

**Comment by JFormDesigner** [comment]
Yes, but have to wait until all dependencies support Java 9.

I know that RichTextFX does not yet support Java 9 (see https://github.com/TomasMikula/RichTextFX/issues/270). And maybe also Flowless.

**Comment by DevCharly** [comment]
Fixed in 1.0: https://github.com/JFormDesigner/markdown-writer-fx/releases
